packages/react-components/react-charts-preview/src/components/Legends/Legends.types.ts

```diff
@@ -1,5 +1,4 @@
 import * as React from 'react';
-import { ITheme, IStyle } from '@fluentui/react/lib/Styling';
 import { IStyleFunctionOrObject } from '@fluentui/react/lib/Utilities';
```

Comment:
remove import for IStyleFunctionOrObject

Reply:
keeping for now, as hover card styles are dependent on this and its breaking on removing this import. Will remove it when the hover card is migrated.